### PR TITLE
Add HostAliases to Glossary page

### PR DESCRIPTION
**Pull request by daminisatya**
#5993 

I added a HostAliases to the glossary page. I request help with adding the right tags for the term, please. Thank you!

### CODE REVIEW

File: content/en/docs/reference/glossary/host-aliases.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: HostAliases
+id: HostAliases
+date: 2019-01-31
+full_link: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-core
+short_description: >
+  A HostAliases is a mapping between the IP address and hostname to be injected into pod's hosts file.
```
Comment (neolit123) [inline content/en/docs/reference/glossary/host-aliases.md]: i would change this to:
> ... into a Pod's `hosts` file.
[1]

File: content/en/docs/reference/glossary/host-aliases.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- operation
+---
+ A HostAliases is a mapping between the IP address and hostname to be injected into pod's hosts file.
```
Comment (neolit123) [inline content/en/docs/reference/glossary/host-aliases.md]: [1]

File: content/en/docs/reference/glossary/host-aliases.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+<!--more-->
+
+[HostAliases](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-corev) is an optional list of hostnames and IP addresses that will be injected into the pod's hosts file if specified. This is only valid for non-hostNetwork pods.
```
Comment (neolit123) [inline content/en/docs/reference/glossary/host-aliases.md]: > into the Pod's `hosts` file

also 
`pods` -> `Pods`

**Comment by daminisatya** [comment]
@neolit123 Thank you for the review. I updated the commit with the review changes.

File: content/en/docs/reference/glossary/host-aliases.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: HostAliases
+id: HostAliases
+date: 2019-01-31
+full_link: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-core
+short_description: >
+  A HostAliases is a mapping between the IP address and hostname to be injected into Pod's hosts file.
```
Comment (neolit123) [inline content/en/docs/reference/glossary/host-aliases.md]: > into a Pod

missing `a` :)
[1]

File: content/en/docs/reference/glossary/host-aliases.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- operation
+---
+ A HostAliases is a mapping between the IP address and hostname to be injected into Pod's hosts file.
```
Comment (neolit123) [inline content/en/docs/reference/glossary/host-aliases.md]: [1]

**Comment by daminisatya** [comment]
Addressed all the changes :) @neolit123

**Comment by cody-clark** [comment]
Hi, @daminisatya! Thanks for adding to the glossary!

Preview: https://deploy-preview-12454--kubernetes-io-master-staging.netlify.com/docs/reference/glossary/?all=true#term-HostAliases

/approve

**Comment by daminisatya** [comment]
@cody-clark Thank you for the merge :)

**Pull request merged**